askama_parser/src/expr.rs

```diff
@@ -270,7 +270,7 @@ impl<'a> Expr<'a> {
                     separated_list0(char(','), ws(move |i| Self::parse(i, level))),
                     Self::Array,
                 ),
-                char(']'),
+                alt((tag(",]"), tag("]"))),
```

Comment:
Hmm, I think we should accomodate any whitespace appearing between `,` and `]`.

Reply:
Good point, I'll add/update the test too